client/source/js/modules/ui/type-selector/type-selector-service.js

```diff
@@ -0,0 +1,36 @@
+/**
+ *  A service with helper functions for calibration.
+ */
```

Comment:
Here is a nice *reason for being* comment :+1: 

Wouldn't hurt to add concrete details about what this guy do like:
1. enables costs options
2. resets calibration

And future methods extending functionality can review this comment and add it as 3. 4. etc